spec/controllers/api/assignments_controller_spec.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe API::AssignmentsController, type: :controller do
+  let(:organization)          { classroom_org                                         }
+  let(:user)                  { classroom_teacher                                     }
+
+  before do
+    GitHubClassroom.flipper[:download_repositories].enable
+    sign_in_as(user)
+
+    @assignment = create(:assignment, organization: organization)
+    @assignment_json = AssignmentSerializer.new(@assignment).to_json
+  end
+
+  describe "GET #index", :vcr do
+    before do
+      get :index, params: { organization_id: organization.slug }
+    end
+
+    it "returns success" do
+      expect(response).to have_http_status(:success)
+    end
+
+    it "returns only one assignment" do
+      expect(json.length).to eql(1)
+    end
+
+    it "returns serialized versions of indivdual assignment" do
+      expect(json.first.to_json).to eq(@assignment_json)
```

Comment:
Nitpick: If you compare hashes instead of JSON test failures would be slightly easier to read